admin/tabs/activity/tracking/ActivityTrackingModel.ts

```diff
@@ -191,30 +175,40 @@ export class ActivityTrackingModel extends HoistModel {
         this.addReaction({
             track: () => {
                 const vals = this.formModel.values;
-                return [vals.startDay, vals.endDay, vals.maxRows];
+                return [vals.startDay, vals.endDay, vals.maxRows, this.filterChooserModel.value];
```

Comment:
It's a common pattern for us to define a getter on this model (eg `get query() {}`) then observe that here, pass it as `body` in `doLoadAsync` fetch call below.